### PR TITLE
Md sa2 29 24/Dmitry Burlyaev/02. GIT. Local

### PR DESCRIPTION
**Pull request by Dmitry-Kitsune**
Homework [02. GIT. Local]

### CODE REVIEW

**Comment by Dmitry-Kitsune** [comment]
Md sa2 29 24/Dmitry Burlyaev/02. GIT. Local

File: .gitignore [hunk 1]
```diff
@@ -10,4 +10,4 @@ __pycache__
 .idea/*
 *.iml
 .project
-.vagrant
+.vagrant
```
Comment (pluhin) [inline .gitignore:13]: Please revert this changes

**Review by pluhin — CHANGES_REQUESTED** [review]
